pivotal-import/README.md

```diff
@@ -17,7 +17,10 @@ In order to run this, you will require a Pivotal account and the ability to sign
 1. Sign up for a Shortcut account at [https://www.shortcut.com/signup](https://www.shortcut.com/signup).
    - **NOTE:** Do not run this importer against an existing Shortcut workspace that already has data you wish to keep.
 1. [Create a new Shortcut API token](https://app.shortcut.com/settings/account/api-tokens) and [export it into your environment](../Authentication.md).
-1. Export your Pivotal project to CSV and save the file to `data/pivotal_export.csv`.
+1. Export your Pivotal project to CSV.
+   - Unarchive the ZIP file provided by Pivotal.
+   - Copy the primary CSV file to `data/pivotal_export.csv`
+   - (Optional) To import your Pivotal story file attachments, ensure you included them when requesting your Pivotal export, and then copy the directories in your Pivotal export that are named after your Pivotal story IDs (which contain their file attachments) into the `data/` folder of this project. This will result in a directory structure like `data/10000/*`, `data/10001/*`, etc.
```

Comment:
At some point, I recommend a followup by including a section that describes the content of the CSV separately from this instruction, to improve clarity